tests/Tokenio/TokenBaseTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Test\Tokenio;
+
+use Google\Protobuf\Internal\RepeatedField;
+use Io\Token\Proto\Common\Address\Address;
+use PHPUnit\Framework\TestCase;
+use Tokenio\Config\TokenCluster;
+use Tokenio\Config\TokenEnvironment;
+use Tokenio\Config\TokenIoBuilder;
+use Tokenio\Util\Strings;
+use Tokenio\Security\UnsecuredFileSystemKeyStore;
+use Io\Token\Proto\Common\Alias\Alias;
+
+abstract class TokenBaseTest extends TestCase
```

Comment:
Is introducing the base test class necessary? We want to keep the implementation approaches similar if possible.

Reply:
That class provides base methods like initSDK(), generateAlias(), generateAddress(), and method to convert RepeatedField to array to perform checks. Those could be moved to some kind of TestUtils.php but to me it looks OK to left it in base class. I can change that if needed.

Reply:
I understand, and I don't think this approach itself has a problem. I was trying to say we try to have implementation consistency across multiple SDKs, so we'd rather not use this "base test class" approach since it is not used else where.

You don't need to do it in this PR, but we do need to change it, to use the TestUtil approach like you mentioned. I'll create a ticket for this and we can do this later.

Reply:
Changed to TestUtil.php